docs/zh-CN/administrator-guide/http-actions/check-reset-rpc-cache.md

```diff
@@ -0,0 +1,46 @@
+---
+{
+    "title": "检查和重置连接缓存",
```

Comment:
You need to add these new docs to sidebar

Reply:
those files already in sidebar